biogeochem/EDPhysiologyMod.F90

```diff
@@ -1079,7 +1117,18 @@ subroutine PlantGrowth( currentSite, currentCohort, bc_in )
     end if
     
     if (EDPftvarcon_inst%evergreen(ipft) == 1)then
-       currentCohort%leaf_md = currentCohort%bl / EDPftvarcon_inst%leaf_long(ipft)
+       if(use_leaf_age==itrue)then
+          if(currentSite%dstatus==1)then
```

Comment:
Is this supposed to apply to the evergreen phenology or the drought-deciduous phenology?  Assuming the former since the parameter file doesn't have that as a thing that's changed. But if so, then won't this have the effect of pushing the leaf further off its target and therefore causing the plant to reallocate more new growth towards leaves?  does this need to be coupled to logic that reduces the target leaf biomass by some factor during the drought period?

Reply:
This is for evergreen and this is based on the observations of more leaf turnover during the drought period. I think it may be reasonable to ajdust the target biomass during the drought period if we have the data.